### ISSUE TITLE
New Faction Jobs Suggestion

### ISSUE THREAD

**Issue by Enderpandi06** [gitrous/origins#81]
Extortion, or collecting protection job money.
Works simple, just like the newspaper job.
You walk to 3-4 doors, and collect "protection" money from the residents and go back to job giver, the job should give around 20-30k to make it worth time, I also say this should be Lopez Cartel excpusive since they are next to apartments and they have more power in the area. Another reason is that Lopez Cartel economy has to be balanced so people can afford the guns in the market without farming for 2 days.

Next suggestion,
Damaging a shop, the leader gives you a baseball bat, or firearm with non-lethal bullets, and you have to break a specific amount of Windows/glasses in the specific shop and go back to leader to get paid. 

Next one: 
Sabotaging. The leader of your gang tells you to move to a gang turf and interact with a specific object(can be a crate, electric box or stuff), after you interact with the object, you go back to gang leader and get paid.

**Comment by xXPirateLordXx** [comment]
the collection money i dont like, other two are cool

**Comment by cI0ak** [comment]
i don't like the money collecting but the other ones are cool. and also, don't make the leader give u a firearm with non lethal rounds, just let him give u a melee and if u don't wanna use that u can use your own gun

**Comment by Enderpandi06** [comment]
Yes, a baseball bat that you have to return after the job is most ideal.
